debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerHelper.java

```diff
@@ -94,7 +96,7 @@ public static void setLogFilesForMining(OracleConnection connection, Scn lastPro
         throw new IllegalStateException("None of log files contains offset SCN: " + lastProcessedScn + ", re-snapshot is required.");
     }
 
-    private static boolean hasNoLogFilesStartingBeforeScn(List<LogFile> logs, Scn scn) {
+    private static boolean hasNoLogFilesStartingBeforeOrAtScn(List<LogFile> logs, Scn scn) {
```

Comment:
I realize the code wasn't written this was prior; however, to align with a recent conversation on a similar topic of method names we had on another PR.  Let's rename this to `hasLogFilesStartingBeforeOrAtScn` and then use the `!` operator in the if-block call to invert the result-set.